### ISSUE TITLE
Research how to support customers with Supply Chain Cyber Security Requirements / SBOM

### ISSUE THREAD

**Issue by mheadd** [cloud-gov/product#1740]
## What are we trying to do

Make it easier for customers to obtain a SBOM or cloud.gov and Federalist (cloud.gov Pages).

## Why are we trying to do it.

[Recent Executive Orders](https://www.whitehouse.gov/briefing-room/presidential-actions/2021/05/12/executive-order-on-improving-the-nations-cybersecurity/) require software providers to provide a Software Bill of Materials (SBOM) to customers. Currently, this is a non-trivial exercise that requires the cloud.gov engineering team.

**Comment by pburkholder** [comment]
We have a dreaft answer on this for a potential customer at [this google doc](https://docs.google.com/document/d/1ftXxYDR81JsOoqFShpxwOo0X0acpPJ4qPW1ah8mj2Ac/edit#)

**Comment by pburkholder** [comment]
Follow on work will be a component of https://github.com/cloud-gov/product/issues/1726.

**Comment by pburkholder** [comment]
I spent closer to 8 hours on this. There are a lot of rabbit holes to fall into. 

I wrote up what I found at https://docs.google.com/document/d/1ftXxYDR81JsOoqFShpxwOo0X0acpPJ4qPW1ah8mj2Ac/edit#heading=h.vmz35ieog6o0 and shipped to requestor and generated example SBOMS for the parts I can:

syft packages bosh/docker-cpi:main > bosh-docker-cpi.txt
syft packages cloudgov/pages-build-container:pages-staging > pages-build-container.txt
syft packages cloudfoundry/cflinuxfs3:latest > cloudfoundry-cflinuxfs3.txt
syft packages dir:./federalist > federalist.txt
syft packages dir:./federalist-builder > federalist-builder.txt
syft packages dir:./federalist-proxy >federalist-proxy.txt

To take a stab at generating packages from releases in BOSH, wrote: 

https://github.com/cloud-gov/cg-scripts/blob/peterb/bosh-to-sbom/bosh-unpack.py

I shipped the PDF and the text files to requestor. So, DONE.
